fs/io_zenfs.cc

```diff
@@ -162,14 +177,16 @@ ZoneFile::ZoneFile(ZonedBlockDevice* zbd, std::string filename,
       fileSize(0),
       filename_(filename),
       file_id_(file_id),
-      nr_synced_extents_(0) {}
+      nr_synced_extents_(0),
+      m_time_(time(0)) {}
```

Comment:
m_time should be 0 after construction(see prev comment), set the modification time when the file is created in stead

Reply:
Done.